src/components/DiagnosticCard/DiagnosticCard.tsx

```diff
@@ -0,0 +1,17 @@
+import {ReactNode} from 'react';
+import cn from 'bem-cn-lite';
+
+import {Card} from '@gravity-ui/uikit';
+
+import './DiagnosticCard.scss';
+
+const b = cn('diagnostic-card');
+
+interface DiagnosticCardProps {
+    children?: ReactNode;
+    className?: string;
+}
+
+export function DiagnosticCard({children, className}: DiagnosticCardProps) {
```

Comment:
Why do you need this component? It seems you can just use a card and set it styles in `TenantOverview`. Or do you plan to use it for cards with metrics?

Reply:
Yes, I'm going to use this component for cards with metrics.